src/media/css/footer.styl

```diff
@@ -45,6 +45,9 @@ $support-nav-height = 55px;
             display: block;
         }
     }
+    .footer-text{
+    color:#717268;
```

Comment:
Please use a predefined colour https://github.com/mozilla/fireplace/blob/master/src/media/css/lib/colors.styl as well as follow the existing CSS indentation practices. A better class name would also be `.page-footer-text {` unless you nest `.footer-text` under `.footer`.